modules/ROOT/pages/rtm-agent-whitelists.adoc

```diff
@@ -71,6 +77,28 @@ In addition, the following endpoints use WebSockets:
 |Region |Name |IP Address
 |*US*|runtime-manager.anypoint.mulesoft.com |18.214.68.14
 |*US*|runtime-manager.anypoint.mulesoft.com |35.174.151.175
+|*US-GOV*|runtime-manager.gov.anypoint.mulesoft.com |52.61.85.119
+|*US-GOV*|runtime-manager.gov.anypoint.mulesoft.com |3.32.47.1
+|*US-GOV*|runtime-manager.gov.anypoint.mulesoft.com |160.1.154.225
+|===
+
+* Whitelist these static IPs in the `US` region to access the `data-authenticator` hosts:
```

Comment:
Allow the following static IPs in the `US` region to access the `data-authenticator` hosts:?